src/app/report/difference-modal/difference-modal.component.ts

```diff
@@ -56,6 +61,10 @@ export class DifferenceModalComponent {
   onClickSave(): void {
     if (this.saveOrDiscardType === 'save') {
       this.saveChangesEvent.next();
+    } else if (this.saveOrDiscardType === 'discard') {
+      this.discardChangesEvent.next();
+    } else {
+      this.toastService.showWarning('Something went wrong.');
```

Comment:
I would make it a little bit more specific. Maybe something like "Could not save/discard changes". This way we can more easily debug when something goes wrong and people using the frontend can get more information about what is going wrong.
What do you think @MatthijsSmets?

Reply:
true. then it's alright